core/src/test/scala-3/cats/derived/FoldableTests.scala

```diff
@@ -2,7 +2,7 @@ package cats.derived
 
 import cats.Foldable
 import cats.derived.all._
-import cats.derived.all.given
+import cats.instances.all._
```

Comment:
Does import given only work with instances defined with `given`, rather than the legacy `implicit` used in cats?

Reply:
I don't know 🤔 - that import shouldn't be necessary anymore

Reply:
👍 Did your cats change to put `Id` instances into given scope get released then?

Reply:
No - but I think _some_ instances were already in implicit scope.
My change was to extend this to all instances.

Reply:
🤦 😂